rules/ref/path/dev.yara

```diff
@@ -0,0 +1,10 @@
+rule dev_path : notable {
+    meta:
+        description = "path reference within /dev"
+    strings:
+        $path = /\/dev\/[a-z\.\-\/]{1,16}/
+        $ignore_null = /\/dev\/nu[l]{1,2}/
```

Comment:
Why match /dev/nul?

Reply:
I don't think it's necessary outside of what I noted in [this](https://github.com/chainguard-dev/bincapz/pull/148#issuecomment-2067720275) comment.

I'll tweak the rule and check the output again.

Reply:
Yea, reverting to `/dev/null` results in this:
```
MED   ref/path/dev                     path reference within /dev                    /dev/nul                                        
                                                                                     /dev/stderr                                     
                                                                                     /dev/stdout
```

Reply:
Also saw this when tweaking the rule:
```
MED   ref/path/dev          path reference within /dev                  /dev/null.    
```